### ISSUE TITLE
various devices cannot be loaded, device cannot be found.  

### ISSUE THREAD

**Issue by tangkong** [pcdshub/device_config#27]
| name                    | class                                       |
|-------------------------|---------------------------------------------|
| cxi_leviton_r51_h       | pcdsdevices.fms.Leviton9                    |
| cxi_leviton_r51_l       | pcdsdevices.fms.Leviton17                   |
| cxi_leviton_r51_t       | pcdsdevices.fms.Leviton1                    |
| cxi_leviton_r52_h       | pcdsdevices.fms.Leviton10                   |
| cxi_leviton_r52_l       | pcdsdevices.fms.Leviton18                   |
| cxi_leviton_r52_t       | pcdsdevices.fms.Leviton2                    |
| cxi_leviton_r53_h       | pcdsdevices.fms.Leviton11                   |
| cxi_leviton_r53_l       | pcdsdevices.fms.Leviton19                   |
| cxi_leviton_r53_t       | pcdsdevices.fms.Leviton3                    |
| cxi_leviton_r54_h       | pcdsdevices.fms.Leviton12                   |
| cxi_leviton_r54_l       | pcdsdevices.fms.Leviton20                   |
| cxi_leviton_r54_t       | pcdsdevices.fms.Leviton4                    |
| cxi_leviton_r56_h       | pcdsdevices.fms.Leviton13                   |
| cxi_leviton_r56_l       | pcdsdevices.fms.Leviton21                   |
| cxi_leviton_r56_t       | pcdsdevices.fms.Leviton5                    |
| cxi_leviton_r57_h       | pcdsdevices.fms.Leviton14                   |
| cxi_leviton_r57_l       | pcdsdevices.fms.Leviton22                   |
| cxi_leviton_r57_t       | pcdsdevices.fms.Leviton6                    |
| cxi_leviton_r58_h       | pcdsdevices.fms.Leviton15                   |
| cxi_leviton_r58_l       | pcdsdevices.fms.Leviton23                   |
| cxi_leviton_r58_t       | pcdsdevices.fms.Leviton7                    |
| cxi_leviton_r59_h       | pcdsdevices.fms.Leviton16                   |
| cxi_leviton_r59_l       | pcdsdevices.fms.Leviton24                   |
| cxi_leviton_r59_t       | pcdsdevices.fms.Leviton8                    |
| feh_setra               | pcdsdevices.fms.Setra5000                   |
| homs_env                | pcdsdevices.rtds_homs.HomsEnvironment       |
| mec_jj_1                | pcdsdevices.slits.JJSlits                   |
| mec_jj_2                | pcdsdevices.slits.JJSlits                   |
| mec_jj_3                | pcdsdevices.slits.JJSlits                   |
| mec_jj_4                | pcdsdevices.slits.JJSlits                   |
| mec_leviton_r60_t       | pcdsdevices.fms.Leviton53                   |
| mec_leviton_r62_t       | pcdsdevices.fms.Leviton51                   |
| mec_leviton_rpci_h      | pcdsdevices.fms.Leviton60                   |
| mec_leviton_rpci_t      | pcdsdevices.fms.Leviton49                   |
| mec_leviton_rs67c_t     | pcdsdevices.fms.Leviton50                   |
| mec_leviton_s60_t       | pcdsdevices.fms.Leviton52                   |
| mfx_leviton_r45_h       | pcdsdevices.fms.Leviton41                   |
| mfx_leviton_r45_p       | pcdsdevices.fms.Leviton45                   |
| mfx_leviton_r45_t       | pcdsdevices.fms.Leviton37                   |
| mfx_leviton_r46_h       | pcdsdevices.fms.Leviton42                   |
| mfx_leviton_r46_p       | pcdsdevices.fms.Leviton46                   |
| mfx_leviton_r46_t       | pcdsdevices.fms.Leviton38                   |
| mfx_leviton_r49_h       | pcdsdevices.fms.Leviton43                   |
| mfx_leviton_r49_p       | pcdsdevices.fms.Leviton47                   |
| mfx_leviton_r49_t       | pcdsdevices.fms.Leviton39                   |
| mfx_leviton_sampletbl_h | pcdsdevices.fms.Leviton44                   |
| mfx_leviton_sampletbl_p | pcdsdevices.fms.Leviton48                   |
| mfx_leviton_sampletbl_t | pcdsdevices.fms.Leviton40                   |
| mfx_lib                 | pcdsdevices.n_axis_device.get_n_axis_device |
| neh_setra               | pcdsdevices.fms.Setra5000                   |
| sq1_lamp                | pcdsdevices.sq1.Sq1                         |
| xcs_leviton_r40_h       | pcdsdevices.fms.Leviton29                   |
| xcs_leviton_r40_p       | pcdsdevices.fms.Leviton33                   |
| xcs_leviton_r40_t       | pcdsdevices.fms.Leviton25                   |
| xcs_leviton_r42_h       | pcdsdevices.fms.Leviton30                   |
| xcs_leviton_r42_p       | pcdsdevices.fms.Leviton34                   |
| xcs_leviton_r42_t       | pcdsdevices.fms.Leviton26                   |
| xcs_leviton_r43_h       | pcdsdevices.fms.Leviton31                   |
| xcs_leviton_r43_p       | pcdsdevices.fms.Leviton35                   |
| xcs_leviton_r43_t       | pcdsdevices.fms.Leviton27                   |
| xcs_leviton_r44_h       | pcdsdevices.fms.Leviton32                   |
| xcs_leviton_r44_p       | pcdsdevices.fms.Leviton36                   |
| xcs_leviton_r44_t       | pcdsdevices.fms.Leviton28                   |
| xpp_leviton_mon_t       | pcdsdevices.fms.Leviton82                   |
| xpp_leviton_r32_t       | pcdsdevices.fms.Leviton83                   |

**Comment by klauer** [comment]
That's a _lot_ of Levitons... 
Doing some digging:
* We never had that in pcdsdevices
* I see no references in the hutches to those
* Given that the author is Cynthia, I wonder if she has a pcdsdevices repo + dev install in her home directory or something... We should probably contact her to fix this up and merged into pcdshub or removed entirely...

**Comment by klauer** [comment]
Found it: `/reg/g/pcds/epics-dev/cymel123/fms/pcdsdevices/pcdsdevices`

**Comment by ZLLentz** [comment]
Yes, cynthia was working on facility monitoring system stuff- seems like our usage of one happi as the "live" happi for all things without dev/prod split will continue to cause conflicts like this.

Peeking at all the other stuff, I think _everything_ on this list is somebody's dev device

**Comment by tangkong** [comment]
I imagine there's no proper policy behind how long something should exist before being cleaned up and pulled into pcdsdevices, but maybe dev devices should have some sort of half life if they're in happi.

**Comment by klauer** [comment]
I think it's not a big deal usually. I feel like there's a certain threshold of maybe 5 or 10 "test" devices after which point there should be some sort of agreement that the underlying implementation gets reviewed/merged in/deleted/whatever.

**Comment by ZLLentz** [comment]
> maybe dev devices should have some sort of half life if they're in happi.

On today's episode of Zach takes everything too literally: every month, we randomly delete half of the invalid devices in happi
